### ISSUE TITLE
[FEATURE] Direct funding from users without an account

### ISSUE THREAD

**Issue by aerugo** [cobudget/cobudget#679]
![image](https://user-images.githubusercontent.com/7785081/162053163-291171ff-cf32-463a-a44b-8e1df7f870e5.png)

Enable direct funding like in #678, but from users without an account. 
Main differences from #678:
 - No selector "From my balance"/"Direct with card"
 - Ask for email (required)
 - Consent to emails from co-creators (pre-ticked, and required if it is a transaction)
 - After payment, give option to enter a name for the funders list or leave blank to stay anonymous

**Comment by aerugo** [comment]
Maybe there should be a prompt to log in here somewhere - for users with accounts who are just not logged in at the moment?

**Comment by tomomiq** [comment]
Good idea, we could add it in the first pop-up and re-direct to this state once they've logged in: 

<img width="833" alt="Screenshot 2022-04-06 at 23 45 12" src="https://user-images.githubusercontent.com/1287179/162077092-7a6d41f6-c8e3-4fce-8857-76197cffa68f.png">
